packages/ecma402-abstract/types/duration.ts

```diff
@@ -0,0 +1,10 @@
+import {LocaleData} from './core'
+
+export interface DurationLocaleData {
+  digitalFormat: {
```

Comment:
Currently only the default numbering system is supported in a locale, this could be converted to something like `Record<string, { digitalFormat: { separator: string } }>` to store the information per-number system.